hydra/_internal/utils.py

```diff
@@ -346,16 +346,18 @@ def add_conf_dir() -> None:
             sys.exit(0)
 
         has_show_cfg = args.cfg is not None
+        has_show_cfg_resolved = args.resolve is not None
         num_commands = (
             args.run
             + has_show_cfg
+            + has_show_cfg_resolved
             + args.multirun
             + args.shell_completion
             + (args.info is not None)
         )
         if num_commands > 1:
             raise ValueError(
-                "Only one of --run, --multirun,  -cfg, --info and --shell_completion can be specified"
+                "Only one of --run, --multirun,  --cfg, --resolve, --info and --shell_completion can be specified"
```

Comment:
I was thinking to make --resolve a free flag that only takes effect if `--cfg` is specified. Not an actual run mode mode.
```
--cfg [job|all|hydra] [--resolve]
```

Reply:
Done in https://github.com/facebookresearch/hydra/pull/1616/commits/c65c50757241e9b338d113c0d6f5b1a5db708184.